kuma/javascript/src/accountsettings/components/subscription-detail.jsx

```diff
@@ -0,0 +1,111 @@
+// @flow
+import * as React from 'react';
+import { useState } from 'react';
+
+import { gettext, interpolate, Interpolated } from '../../l10n.js';
+import { formatDate, formatMoney } from '../../formatters.js';
+
+import { type SubscriptionData } from '../../hooks/useSubscriptionData.js';
+import CancelSubscriptionForm from './cancel-subscription-form.jsx';
+
+type Props = {
+    contributionSupportEmail: string,
+    handleDeleteSuccess: Function,
```

Comment:
React naming convention tends to be prefixing the prop side of events with `on` whereas `handle` is used for the concrete function actually handling the event. So specifically I'd recommend renaming this do `onDeleteSuccess`.